src/Form/Control/Upload.php

```diff
@@ -0,0 +1,77 @@
+<?php
+namespace atk4\filestore\Form\Control;
+
+class Upload extends \atk4\ui\Form\Control\Upload 
```

Comment:
new file, isn't solved somewhere else?

Reply:
Sure, just moved the old Upload.php into the new folder structure

Reply:
moved, really? I se a new file and no deletion...

Reply:
it's moved